src/components/Icon/IconTextkernel/IconTextkernel.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import IconBase, { Props as BaseProps } from '../IconBase';
+
+interface Props extends Omit<Omit<BaseProps, 'children'>, 'viewbox'> {}
```

Comment:
Should we remove `viewbox`? It can be used like `IconBase` prop  ?

Reply:
I don't understand what do you mean. 'Viewbox' is hard coded based on the svg path. You should not change it through the props. You can however change the size of the icon, using `size`